packages/lit-html/src/test/directives/async-replace_test.ts

```diff
@@ -213,9 +213,14 @@ suite('asyncReplace', () => {
     const delay = (delay: number) =>
       new Promise((res) => setTimeout(res, delay));
 
-    render(component(generator(delay(20), 'slow')), container);
-    render(component(generator(delay(10), 'fast')), container);
-    await delay(30);
+    const slowDelay = delay(20);
+    const fastDelay = delay(10);
+
+    render(component(generator(slowDelay, 'slow')), container);
+    render(component(generator(fastDelay, 'fast')), container);
+
+    await slowDelay;
+    await delay(10);
```

Comment:
I think you can just await a microtask here instead of a setTimeout

Reply:
One microtask may not be enough to get behind the yields in the generator and the awaits in the directive. Plus we're working around fake promise timing voodoo on IE11. Could change it to rAF, but delay was sitting there.